### ISSUE TITLE
Latest Release breaks compatibility with Python 3.7

### ISSUE THREAD

**Issue by JacksonBurns** [yu9824/kennard_stone#6]
The latest changes in v2 are no longer compatible with Python 3.7. Run this example in an environment with Python 3.7 installed to reproduce the error:
```python
>>> from kennard_stone import train_test_split
>>> import numpy as np

>>> train_test_split(np.array([1,2,3]).reshape(-1, 1))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jackson/mambaforge/envs/py37/lib/python3.7/site-packages/kennard_stone/kennard_stone.py", line 142, in train_test_split
    train, test = next(cv.split(X=arrays[0]))
  File "/home/jackson/mambaforge/envs/py37/lib/python3.7/site-packages/sklearn/model_selection/_split.py", line 1600, in split
    for train, test in self._iter_indices(X, y, groups):
  File "/home/jackson/mambaforge/envs/py37/lib/python3.7/site-packages/kennard_stone/kennard_stone.py", line 75, in _iter_indices
    indexes = ks.get_indexes(X)[0]
  File "/home/jackson/mambaforge/envs/py37/lib/python3.7/site-packages/kennard_stone/kennard_stone.py", line 194, in get_indexes
    X=X, lst_idx_selected=lst_idx_selected, idx_remaining=idx_remaining
  File "/home/jackson/mambaforge/envs/py37/lib/python3.7/site-packages/kennard_stone/kennard_stone.py", line 211, in _sort
    sum(lst_idx_selected, start=[])
TypeError: sum() takes no keyword arguments
```

This change in the call to sum should be reverted, or Python 3.7 should be dropped from the list of supported Python versions.

**Comment by yu9824** [comment]
Thank you very much for your report. I had missed the bug.

I've fixed the bug and performed `pytest` on each version.
https://github.com/yu9824/kennard_stone/actions/runs/4798295831

I will release the fixed version soon.
Best regards

**Comment by yu9824** [comment]
I have released a modified version (v2.0.1) on both PyPI and Anaconda.
